apps/train_loc/test/integration/integration_test.exs

```diff
@@ -68,9 +68,4 @@ defmodule TrainLoc.IntegrationTest do
         :timeout
     end
   end
-
-  @tag :integration
-  test "one minute of messages with 11 up-to-date vehicles and one conflict" do
-    run_test(TrainLoc.IntegrationTest.Scenarios.OneMinute)
```

Comment:
question: what module is this referencing? can we delete that module as well?

Reply:
Removed `Scenarios.OneMinute`. That was only referenced from the removed IntegrationTest unit test.